tools/ds-identify

```diff
@@ -331,7 +389,7 @@ read_virt() {
 
 is_container() {
     case "${DI_VIRT}" in
-        container-other|lxc|lxc-libvirt|systemd-nspawn|docker|rkt) return 0;;
+        container-other|lxc|lxc-libvirt|systemd-nspawn|docker|rkt|jail) return 0;;
```

Comment:
but if I'm already touching a code path where this function is called, and adapting it to FreeBSD, then i figured I'd make is_container work on (and for!) FreeBSD

Reply:
can we make that bsd-jail ?